apps/condo/domains/ticket/access/TicketChange.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Generated by `createschema ticket.TicketChange 'ticket:Relationship:Ticket:CASCADE;'`
+ */
+
+async function canReadTicketChanges ({ authentication: { item: user } }) {
+    if (!user) return false
+    if (user.isAdmin) return {}
+    return {
+        // TODO(codegen): write canReadTicketChanges logic!
+    }
+}
+
+async function canManageTicketChanges ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return false
+    if (user.isAdmin) return true
```

Comment:
I think we don't need to create/update it by anyone instead of inner logic.
just `return false`